src/labels.ts

```diff
@@ -78,6 +78,9 @@ export const removeBackportLabelsFromPrsTargetingReleaseBranches = async () => {
 
 // given a list of PRs, removes the backport/* labels from them
 export const removeBackportLabelsFromPrs = (prs) => {
+  if (prs === undefined) {
+    throw new Error("removeBackportLabelsFromPrs called with undefined");
+  }
```

Comment:
Hmm, if you add a type to `prs`, typescript could likely statically determine whether this function could ever be called with `undefined`.